search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -272,6 +276,13 @@
                            :distribution (not-empty (map related-url->distribution related-urls))
                            :landingPage (landing-page related-urls)
                            :language  [LANGUAGE_CODE]
+                           :creator (not-empty (keep :Creator collection-citations))
```

Comment:
Does the JSON validate successfully here: https://labs.data.gov/dashboard/validate ?

I think Erich mentioned that there's a way to provide an expanded JSON schema and link to that. Should we be doing that instead?

Reply:
I'm not sure of the answer to data.nasa.gov. When I submit the example opendata.json file I used for the above output to the validate endpoint I get a '100% Valid!' response. Looks like we can add anything we want to the response and data.gov is smart enough to place them in the right spots. We can also add the graphic-preview-file field and it will display the image like in the NOAA example.

Reply:
> We can also add the graphic-preview-file field and it will display the image like in the NOAA example.

Awesome.